quarkus/pom.xml

```diff
@@ -67,7 +67,8 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
 
-        <surefire-plugin.version>3.0.0-M6</surefire-plugin.version>
+        <!-- Needs to be aligned with Quarkus, see e.g. https://github.com/quarkusio/quarkus-quickstarts/blob/2.7.6.Final/getting-started-async/pom.xml#L14 -->
+        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
```

Comment:
Not doing this for the Operator for now, otherwise we'd be affected by [SUREFIRE-1926](https://issues.apache.org/jira/browse/SUREFIRE-1926).